model/convert/service_event.go

```diff
@@ -398,7 +401,17 @@ func convertServiceEventEpochRecover(event flow.Event) (*flow.ServiceEvent, erro
 
 	cdcDKGKeys, err := getField[cadence.Array](fields, "dkgPubKeys")
 	if err != nil {
-		return nil, fmt.Errorf("failed to decode EpochRecover event: %w", err)
+		return nil, fmt.Errorf("failed to decode EpochCommit event: %w", err)
+	}
+
+	cdcDKGGroupKey, err := getField[cadence.String](fields, "dkgGroupKey")
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode EpochCommit event: %w", err)
+	}
+
+	cdcDKGIndexMap, err := getField[cadence.Dictionary](fields, "dkgIdMapping")
+	if err != nil {
+		return nil, fmt.Errorf("failed to decode EpochCommit event: %w", err)
```

Comment:
In those three error messages, we are talking about the `EpochCommit` event, while most other error messages are mentioning the `EpochRecover` event. I think we should make it consistent:
* In my opinion,  it would be fine to just talk about the `EpochRecover` in all error messages. This is because some fields (like the epoch counter) are shared between the `EpochSetup` and `EpochCommit` events. 
* Furthermore, I think there is little benefit in distinguishing the fields in the `EpochRecover` by whether they belong into the `EpochSetup` or `EpochCommit` event. In the end what matters is that we couldn't decode some specific field (everything beyond is an implementation details that is not very useful to expose in the error message)